### ISSUE TITLE
RexRay S3FS docker plugin is not requesting the correct permissions

### ISSUE THREAD

**Issue by matthid** [codedellemc/rexray#896]
# Summary

See comment in https://hub.docker.com/r/rexray/s3fs/

Basically, when installing the version 0.9.1 doesn't ask for all the required docker permissions.

This makes the plugin unusable

It creates something like
```
fuse: device not found, try 'modprobe fuse' first
```

in the docker logs and

```
docker: Error response from daemon: error while mounting volume '/var/lib/docker/plugins/7ad9c6577429f09685e9078dd308a8599c78fadb1913462a6f0c1d9f5ce5e429/rootfs': VolumeDriver.Mount: {"Error":"error executing xcli"}.
```

when running from the command line.

## Version

0.9.1 from dockerhub (https://hub.docker.com/r/rexray/s3fs/)

## Expected Behavior
```
$ docker plugin install rexray/s3fs:0.9.1 S3FS_ACCESSKEY=<key> S3FS_SECRETKEY=<secret>
Plugin "rexray/s3fs:0.9.1" is requesting the following privileges:
 - network: [host]
 - mount: [/dev]
 - allow-all-devices: [true]
 - capabilities: [CAP_SYS_ADMIN]
Do you grant the above permissions? [y/N]
```

The plugin should work.

## Actual Behavior

```
$ docker plugin install rexray/s3fs:0.9.1 S3FS_ACCESSKEY=<key> S3FS_SECRETKEY=<secret>
Plugin "rexray/s3fs:0.9.1" is requesting the following privileges:
 - network: [host]
Do you grant the above permissions? [y/N]
```
It doesn't.

## Steps To Reproduce
Please list the steps to reproduce the issue in this section.

1. `docker plugin install --grant-all-permissions rexray/s3fs:0.9.1`
2. try to use the plugin, `docker volume list` works everything else doesn't

## Configuration Files
Not applicable?

## Logs
It's actually not trivial to get Docker daemon logs on windows with MobyLinuxVM (HyperV).
But I can send a log file if required (I don't think it is)

**Comment by matthid** [comment]
The workaround is to use `0.9.0` for now.

**Comment by codenrhoden** [comment]
Closing as duplicate of #891.

Fix is committed, will be released in 0.9.2.
